pom.xml

```diff
@@ -66,6 +66,11 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>jackson2-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>snakeyaml-api</artifactId>
+      <version>2.1-110.v4960f4074978</version>    
```

Comment:
This is the incremental version of snakeyaml-api, needs to be updated when https://github.com/jenkinsci/snakeyaml-api-plugin/pull/94 is merged and released

Reply:
https://github.com/jenkinsci/snakeyaml-api-plugin/pull/94